apps/demo/src/app/cart/components/cart/cart.component.scss

```diff
@@ -2,9 +2,48 @@
 @import 'theme';
 
 .demo-cart {
```

Comment:
And use `:host` to apply these styles to the root element?

Reply:
yep, sounds good